src/main/resources/schemas/nhinc/common/configAdmin.xsd

```diff
@@ -8,6 +8,7 @@
             <xsd:element name="userInfo" type="nccommon:UserType" />
             <xsd:element name="configInstance" type="xsd:string" />
             <xsd:element name="authMethod" type="xsd:string" />
+			<xsd:element name="hashToken" type="xsd:string" />
```

Comment:
Please move hashToken out of the assertion info.  It's specific to the certificate store, so should be in the certificate requests (edit/delete/import).